tests/macros/defaultapisidebar.test.js

```diff
@@ -0,0 +1,359 @@
+/**
+ * @prettier
+ */
+const { JSDOM } = require('jsdom');
+
+const {
+    beforeEachMacro,
+    describeMacro,
+    itMacro,
+    lintHTML
+} = require('./utils');
+
+/**
+* Load all the fixtures.
+*/
+const fs = require('fs');
+const path = require('path');
+const subpagesFixturePath = path.resolve(__dirname, 'fixtures/defaultapisidebar/subpages.json');
+const subpagesFixture = JSON.parse(fs.readFileSync(subpagesFixturePath, 'utf8'));
+const commonl10nFixturePath = path.resolve(__dirname, 'fixtures/defaultapisidebar/commonl10n.json');
+const commonl10nFixture = fs.readFileSync(commonl10nFixturePath, 'utf8');
+const commonL10nJSON = JSON.parse(commonl10nFixture);
+const groupDataFixturePath = path.resolve(__dirname, 'fixtures/defaultapisidebar/groupdata.json');
+const groupDataFixture = fs.readFileSync(groupDataFixturePath, 'utf8');
```

Comment:
I’d prefer if this used `readFixture(…)` from https://github.com/mdn/kumascript/pull/1093.